pkg/querier/queryrange/limits.go

```diff
@@ -366,11 +366,13 @@ func WeightedParallelism(
 	tsdbMaxQueryParallelism := l.TSDBMaxQueryParallelism(user)
 	regMaxQueryParallelism := l.MaxQueryParallelism(user)
 	if tsdbMaxQueryParallelism+regMaxQueryParallelism == 0 {
+		level.Info(util_log.Logger).Log("msg", "querying disabled for tenant")
 		return 0
 	}
 
 	// query end before start would anyways error out so just short circuit and return 1
 	if end < start {
+		level.Warn(util_log.Logger).Log("msg", "query end time before start, letting downstream code handle it gracefully", "start", start, "end", end)
```

Comment:
Do we include the tenant in these logs? And how will we be able to identify which query caused this warning log?

Reply:
yeah, good callout! I have made some changes to log `orgID` and `traceID` with the logs.